pkg/metaclient/kvclient/client.go

```diff
@@ -0,0 +1,30 @@
+package kvclient
+
+import (
+	"github.com/hanfei1991/microcosm/pkg/metaclient"
+	"github.com/hanfei1991/microcosm/pkg/metaclient/kvclient/etcdkv"
+	"github.com/hanfei1991/microcosm/pkg/metaclient/namespace"
+)
+
+// etcdKVClient is the implement of kv interface based on etcd
+// Support namespace isolation and all kv ability
+// etcdImpl -> kvPrefix+etcdClientCloser -> etcdKVClient
+type etcdKVClient struct {
+	metaclient.Closer
+	metaclient.KV
+	leaseID string
+}
+
+func NewEtcdKVClient(config *metaclient.Config, leaseID string) (metaclient.KVClient, error) {
```

Comment:
the name is leaseID, it makes me think that after the lease is expired, the namespace will be reset 🤔

Reply:
I will change to tenantID.